README.md

```diff
@@ -1,10 +1,143 @@
+   ```
+4. Inside `./app.module.ts`, add `CUSTOM_ELEMENTS_SCHEMA` to @NgModule.schemas. This enables custom elements in your Angular application.
+   ```TypeScript
+      import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+
+      @NgModule({
+        declarations: [
+           AppComponent,
+           ...
+           ],
+         imports: [
+           ...
+          ],
+         schemas: [CUSTOM_ELEMENTS_SCHEMA]
+       })
+   ```
+5. Example usage of `lf-checklist`, with the implementation coming from the CDN and the types provided in the `@laserfiche/types-lf-ui-components` NPM package.
+
+   In a html file, 
+   ```html
+   <lf-checklist #myChecklist (checklistChanged)="onChecklistChangedAsync($event)"></lf-checklist>
+   <!-- checklistChanged is a HTML custom event -->
+   ```
+   To add a handle for this element, in the `.ts` file,
+   ```ts
+   import { AfterViewInit, Component, ViewChild, ElementRef} from '@angular/core';
+   // we can directly import types from the NPM package
+   import {LfChecklistComponent, Checklist} from '@laserfiche/types-lf-ui-components';  
+  
+   @Component({
+     selector: 'app-attachments',
+     templateUrl: './attachments.component.html',
+     styleUrls: ['./attachments.component.css'],
+   })
+   export class ExampleApp implements AfterViewInit {
+
+     // allows access to the LfChecklistElement in the html above, available starting in ngAfterViewInit hook
+     @ViewChild('myChecklist') checklist!: ElementRef<LfChecklistComponent>; 
+
+     // type Checklist is the type of the custom element imported from the NPM package
+     currentChecklist: Checklist[]; 
+
+     // events for the elements are received as CustomEvents
+     onClick(event: CustomEvent<Checklist>) {
+       const checklist = event.detail;   // data will be stored in the detail property
+     }
+
+     ngAfterViewInit() {
+        // the element's view is initialized after ngAfterViewInit
+       this.checklist.nativeElement.initAsync(...)
+     }
+   }
+   ```
+## NPM library in Angular Projects
+
+1. `npm install @laserfiche/lf-ui-components`
+2. Inside `./app.module.ts`, import the Angular modules. For example, we want to use the <lf-checklist> element:
+   ```TypeScript
+      import { NgModule } from '@angular/core';
+      import { LfChecklistModule } from '@laserfiche/lf-ui-components/lf-checklist';
+
+      @NgModule({
+        declarations: [
+           AppComponent,
+           ...
+           ],
+         imports: [
+           LfChecklistModule,
+           BrowserAnimationsModule
+          ]
+       })
+   ```
+3. Import the style sheet from the npm package by adding this line in `styles.css` in your angular application.
+```css
+@import '~@laserfiche/lf-ui-components/cdn/lf-laserfiche-lite.css';
+```
+4. the same usage of using `lf-checklist`, except this time, both the implementation and the typing are provided via `@laserfiche/lf-ui-components/lf-checklist`
+
+In a html file, 
```

Comment:
I'm not clear on what the difference is between these lines and the ones above... They seem to both be in an Angular project. We only need one